docs/community/highlights.md

```diff
@@ -6,6 +6,12 @@ keywords:
   - highlights
 ---
 
-The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions whether it's an article you wrote for [Codecademy's Docs website](https://www.codecademy.com/pages/contribute-docs), revising one of the lessons on [freeCodeCamp's curricula](https://contribute.freecodecamp.org/#/index?id=learning-platform), or creating a tutorial for people who use [Audacity](https://support.audacityteam.org/community/contributing/tutorials).
+The Highlights feature is what we like to call the "glitzy pepperoni" of your OpenSauced profile. It's the place where you can display your favorite open source contributions, share the story, and inspire others to join you in your open source journey. For maintainers, it's a great way to showcase your project, the issues that need support, and attract new contributors.
 
-Eager to get started? Check out[the Effectively Highlights section in our free "Intro to Open Source" course](https://github.com/open-sauced/intro/blob/main/06-the-secret-sauce.md#effectively-highlight-your-contributions) to learn more.
+There are currently three types of highlights you can add to your profile: a Dev.to blog post, a pull request, or an issue.
```

Comment:
[Proofreading] I think formatting them as a numbered list makes it easier for users and contributors to read. 
```suggestion
There are currently three types of highlights you can add to your profile: 1. a Dev.to blog post
2. a Pull request
3. a issue
```